lib/write-translations.js

```diff
@@ -9,15 +9,16 @@
 
 /* generate the i18n/en.json file */
 
-const CWD = process.cwd();
+const traverse = require('babel-traverse').default;
+const babylon = require('babylon');
 const fs = require('fs-extra');
-const mkdirp = require('mkdirp');
 const glob = require('glob');
+const mkdirp = require('mkdirp');
+const nodePath = require('path');
```

Comment:
Is there a reason why we use `nodePath` instead of `path` ? I saw `path` being used in another file. Is there conflicting name here ?

Reply:
Yep, you're right